src/api/middleware/validators/application.js

```diff
@@ -0,0 +1,45 @@
+const { body } = require("express-validator");
+
+const { useExpressValidators } = require("../errorHandler");
+const ValidationReasons = require("./validationReasons");
+const { checkDuplicatedEmail } = require("./validatorUtils");
+const companyApplicationConstants = require("../../../models/constants/CompanyApplication");
+const companyConstants = require("../../../models/constants/Company");
+const accountConstants = require("../../../models/constants/Account");
```

Comment:
Please change these to PascalCase, (i.e CompanyApplicationConstants, CompanyConstants, AccountConstants) since they are objects containing constants